spring-boot/src/main/java/org/springframework/boot/jta/narayana/NarayanaProperties.java

```diff
@@ -108,6 +110,11 @@
 	private List<String> expiryScanners = new ArrayList<String>(Collections.singletonList(
 			"com.arjuna.ats.internal.arjuna.recovery.ExpiredTransactionStatusManagerScanner"));
 
+	/**
+	 * Pool setting.
+	 */
+	private Map<String, String> pool = new LinkedHashMap<String, String>();
```

Comment:
What type of properties can be used here? Maybe they should be expanded to the separate entries rather than kept in a map?